celery/backends/base.py

```diff
@@ -644,8 +644,9 @@ def set_chord_size(self, group_id, chord_size):
     def fallback_chord_unlock(self, header_result, body, countdown=1,
                               **kwargs):
         kwargs['result'] = [r.as_tuple() for r in header_result]
-        queue = body.options.get('queue', getattr(body.type, 'queue', None))
-        priority = body.options.get('priority', getattr(body.type, 'priority', 0))
+        body_type = body.get('type', None)
```

Comment:
this change need to be unit test covered

Reply:
Will add unit tests and fix current failing one.

Oddly enough the auto unit tests here fail, yet running on windows 10 via cli the same test passes, even odder if I debug through the test it fails.

I will attempt to resolve all this tonight.

Reply:
@auvipy Unit tests added to existing test_chord_unlock_queue test. If you want a separate test for this I can split it out. Just let me know!